### DISCUSSION TITLE
help me please

### DISCUSSION THREAD

**Discussion by MantraxStudios** [NVIDIA-Omniverse/PhysX#289] [Q&A] [answered]
Hi I need your help I am trying to use physx5 in a game engine I am making I am compiling with makefile on windows and I am importing the libraries and I am only using the #include <PxPhysicsAPI.h> just to see if it will compile but it gives me these errors

y estas librerias
PHYSICS_LIBS_FILES = -lLowLevel_static_64 -lLowLevelAABB_static_64 -lLowLevelDynamics_static_64 -lPhysX_64 -lPhysXCharacterKinematic_static_64 -lPhysXCommon_64 -lPhysXCooking_64 -lPhysXExtensions_static_64 -lPhysXFoundation_64 -lPhysXPvdSDK_static_64 -lPhysXTask_static_64 -lPhysXVehicle_static_64 -lPhysXVehicle2_static_64 -lPVDRuntime_64 -lSceneQuery_static_64 -lSimulationController_static_64 -lSnippetRender_static_64 -lSnippetUtils_static_64

![image](https://github.com/NVIDIA-Omniverse/PhysX/assets/85538374/7ab652a8-53cb-49ec-b6d1-7826a5f50c62)

**Comment by MantraxStudios** [comment, marked as answer]
![image](https://github.com/NVIDIA-Omniverse/PhysX/assets/85538374/ad053123-7cd8-4896-a67a-ea5bbf430fbe)

now I have this error in several files

**Comment by MantraxStudios** [comment]
I managed to compile without errors but it says that there are bugs in the libraries I am using MingW and I do not know how to do that because when I compile with visual studio is only for MSVC I was told please someone explain me thanks

**Comment by vreutskyy** [comment]
Hi @GrstudiosGamesPro 
I may try to help you, but could you better explain what you're doing, step by step, and what doesn't work for you?
Cheers

**Reply by MantraxStudios** [reply]
hello thank you very much I fixed it the error was because the .libs are not compatible with the MingW compiler they are only compatible with the MSVC compiler so my error now is that I am missing a lib called FastXML_64_Static.lib and I don't know where to generate it from please help me

**Reply by vreutskyy** [reply]
As I understand, FastXML on Windows is linked as the source code into PhysXExtensions library. That's why there's no separate FastXML library.

**Reply by MantraxStudios** [reply]
but if it is linked what I don't understand is why it asks for FastXML_64_static.lib

**Reply by vreutskyy** [reply]
how do you generate PhysX's projects?

**Reply by MantraxStudios** [reply]
I am generating them with visual studio 2022 in Debug mode and in Release mode and in both I am asked for that library.

**Reply by vreutskyy** [reply]
no, I mean how you generate projects, not build them. do you use generate_projects.bat? which option?

**Reply by MantraxStudios** [reply]
yes with the option 1

**Reply by vreutskyy** [reply]
isn't option 1 for Linux, not for Windows?

**Reply by MantraxStudios** [reply]
![image](https://github.com/NVIDIA-Omniverse/PhysX/assets/85538374/67cb1d49-7a15-43ca-8130-1ef465808c86)

**Reply by vreutskyy** [reply]
aa, it shows me different options. this one seems correct. then, do you get those errors while building PhysX? or while building your project linking PhysX?
